src/EditorFeatures/CSharp/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -36,7 +36,7 @@ internal class CSharpCodeCleanupService(ICodeFixService codeFixService, IDiagnos
                 IDEDiagnosticIds.RemoveUnnecessaryParenthesesDiagnosticId,
                 IDEDiagnosticIds.AddRequiredParenthesesDiagnosticId),
             new DiagnosticSet(AnalyzersResources.Add_accessibility_modifiers,
```

Comment:
Arguably the resource could be updated too. I'm guessing this is what we show in "here's what code cleanup can do" -- so maybe it could say both, but I can also imagine if the common case is this does an add, maybe it's best to keep the text simpler.

tl;dr: feel free to do what you want here.

Reply:
there are actually two resources.  so you get the precise message :)